test/unit/calculators/benefit_cap_calculator_configuration_test.rb

```diff
@@ -134,6 +135,49 @@ class BenefitCapCalculatorConfigurationTest < ActiveSupport::TestCase
           end
         end
       end
+
+      context "location of user" do
+        context "lives outside of London" do
+          setup do
+            GdsApi::Imminence.any_instance.stubs(:areas_for_postcode).returns(
+              OpenStruct.new(
+                code: 200,
+                results: [
+                  {
+                    slug: "birmingham-city-council",
+                    country_name: "England"
+                  }
+                ]
+              )
+            )
+          end
+          should "return false" do
+            assert_equal false, @config.is_london?("B1%201PW")
+          end
+        end
+        context "lives in London" do
+          setup do
+            GdsApi::Imminence.any_instance.stubs(:areas_for_postcode).returns(
+              OpenStruct.new(
+                code: 200,
+                results: [
+                  {
+                    slug: "camden-borough-council",
+                    country_name: "England"
+                  },
+                  {
+                    slug: "london",
+                    country_name: "England"
+                  }
+                ]
+              )
+            )
+          end
+          should "return true" do
+            assert_equal true, @config.is_london?("WC2B%206SE")
+          end
+        end
+      end
```

Comment:
Minor: I think it would be better to avoid `any_instance` and stub directly on `Services.imminence_api`.